piksi_tools/console/generate_settings.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 
-from __future__ import absolute_import, print_function
```

Comment:
hmm, I don't think it has a direct effect in this case because the code already follows these guidelines. E.g., `from __future__ import division` could make a real functional difference.

But on the other hand without these imports somebody could make some development under Python 2.7 environment that wouldn't be 3.x compatible so maybe it is safer to retain these after all.